api/src/main/java/org/killbill/billing/subscription/api/SubscriptionBaseInternalApi.java

```diff
@@ -52,6 +52,9 @@ public SubscriptionBase createSubscription(UUID bundleId, PlanPhaseSpecifier spe
     public List<SubscriptionBase> createBaseSubscriptionWithAddOns(UUID bundleId, Iterable<EntitlementSpecifier> entitlements, DateTime requestedDateWithMs,
```

Comment:
This extra internal method can probably go away (you might already be working on it).